src/realm/exec/CMakeLists.txt

```diff
@@ -31,12 +31,19 @@ set_target_properties(RealmBrowser PROPERTIES
 )
 target_link_libraries(RealmBrowser Storage)
 
-add_executable(Realm2JSON EXCLUDE_FROM_ALL realm2json.cpp )
-set_target_properties(Realm2JSON PROPERTIES
-    OUTPUT_NAME "realm2json-6"
-    DEBUG_POSTFIX ${CMAKE_DEBUG_POSTFIX}
-)
-target_link_libraries(Realm2JSON Storage)
+if(NOT CMAKE_SYSTEM_NAME MATCHES "^Windows")
+    add_executable(Realm2JSON realm2json.cpp )
+    set_target_properties(Realm2JSON PROPERTIES
+        OUTPUT_NAME "realm2json-6"
+        DEBUG_POSTFIX ${CMAKE_DEBUG_POSTFIX}
+    )
+    target_link_libraries(Realm2JSON Storage)
+
+    install(TARGETS
+            Realm2JSON
+            COMPONENT runtime
```

Comment:
Do we want to put this command-line tool in the runtime package? @fealebenpae, any thoughts?

Reply:
Happy to put it elsewhere, just figured it should go wherever the StateApplier tool is since the fuzz tests are really the only things consuming those executables I think

Reply:
Looking at the contents of that archive, it makes sense to me too:
➜  build git:(RCORE-430) ✗ tar -tzvf realm-Debug-v10.2.0-37-g0f63aa673-Darwin-runtime.tar.gz
drwxr-xr-x  0 jreams staff       0 Dec  4 16:14 bin/
-rwxr-xr-x  0 jreams staff 29722536 Dec  3 16:06 bin/realm-apply-to-state-dbg
-rwxr-xr-x  0 jreams staff 20749816 Dec  3 16:06 bin/realm-dump-dbg
-rwxr-xr-x  0 jreams staff 30659704 Dec  3 16:06 bin/realm-encryption-transformer-dbg
-rwxr-xr-x  0 jreams staff 29946800 Dec  3 16:06 bin/realm-server-index-dbg
-rwxr-xr-x  0 jreams staff 36683896 Dec  3 16:06 bin/realm-sync-worker-dbg
-rwxr-xr-x  0 jreams staff 30593736 Dec  3 16:06 bin/realm-vacuum-dbg
-rwxr-xr-x  0 jreams staff 29882800 Dec  3 16:06 bin/realm-verify-server-file-dbg